internal/commands/user/input_test.go

```diff
@@ -0,0 +1,176 @@
+package user
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+
+	"github.com/Netflix/go-expect"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+func TestResolveUsersInputs(t *testing.T) {
+	testUsers := []realm.User{
+		{
+			ID: "user-1",
+			Identities: []realm.UserIdentity{
```

Comment:
can we inline the `Identities` declaration to a one-liner, just to reduce some line heft here?